cookbooks/clearwater/recipes/security.rb

```diff
@@ -40,7 +40,7 @@
   recursive true
   notifies :run, "ruby_block[get-secret-key]", :immediately
   notifies :create, "template[/etc/apt/apt.conf.d/45_clearwater_repo]", :immediately
-  only_if { URI(node[:clearwater][:repo_server]).scheme == "https" }
+  only_if { node[:clearwater][:repo_servers].any? do |rs| URI(rs).scheme == "https" end }
```

Comment:
If you prefer you could make the outer `{}` into `do ... end` which saves on line length (and below).